document_merge_service/settings.py

```diff
@@ -97,6 +99,17 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
 }
 
 
+# CORS
+CORS_ORIGIN_ALLOW_ALL = env.bool("CORS_ORIGIN_ALLOW_ALL", False)
+CORS_ORIGIN_REGEX_WHITELIST = (
+    (
+        r"{0}".format(
+            env.str("CORS_ORIGIN_REGEX_WHITELIST", r"^(http://)?127\.0\.0\.1:\d{4}$")
```

Comment:
Shouldn't we make this an `env.list`, so multiple values can be configured?

The localhost ones could probably be hardcoded above

Reply:
Done